Podcast/RecommendedEpisodesOuterTableViewCell.swift

```diff
@@ -15,6 +15,8 @@ protocol RecommendedEpisodesOuterTableViewCellDataSource {
 
 protocol RecommendedEpisodesOuterTableViewCellDelegate{
     func recommendedEpisodesOuterTableViewCell(cell: RecommendedEpisodesOuterTableViewCell, didSelectItemAt indexPath: IndexPath)
+    func recommendedEpisodesOuterTableViewCellPushTagViewController(tagViewController: TagViewController)
+    func recommendedEpisodesOuterTableViewCellShowActionSheet(actionSheetViewController: ActionSheetViewController)
```

Comment:
table view cell should not create an action sheet controller, this should be more like: recommendedEpisodesOuterTableViewCellDidTapShowActionSheetButton()